src/styles/_reset.scss

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
I suggest to use `normalize` instead of `reset`. You can find code [here](https://necolas.github.io/normalize.css/latest/normalize.css)

Reply:
Sure, I'll switch to normalize.css. Thanks for the suggestion!